roles/keycloak_realm/tasks/manage_client_roles.yml

```diff
@@ -3,10 +3,12 @@
     name: "{{ item }}"
     realm: "{{ client.realm }}"
     client_id: "{{ client.name }}"
+    description: "{{ item.description | default(omit) }}"
```

Comment:
this was meant to be `client.description` ? `roles` is a list of strings, if you want to allow name and description for roles, it has to be modified and documented as list of dicts; see: https://github.com/ansible-middleware/keycloak/blob/0fdd89d571bc5bbbd2998884563c94bef640d544/playbooks/keycloak-realm.yml#L53
and then the above become name: "{{ item.name }}" and description: "{{ item.description | default(omit) }}"